apps/meteor/app/2fa/client/overrideMeteorCall.ts

```diff
@@ -20,7 +20,7 @@ const callWithTotp =
 				(error as { toastrShowed?: true }).toastrShowed = true;
 				dispatchToastMessage({
 					type: 'error',
-					message: t('Invalid_two_factor_code'),
+					message: t('Invalid_password'),
```

Comment:
```suggestion
					message: twoFactorMethod === 'password' ? t('Invalid_password') : t('Invalid_two_factor_code'),
```

Reply:
@dougfabris changes are made in new commit as per suggested